ios/extensions/UIApplication.swift

```diff
@@ -9,6 +9,22 @@ import Foundation
 import UIKit
 
 public extension UIApplication {
+  var activeWindow: UIWindow? {
+    if #available(iOS 13.0, *) {
+      for scene in connectedScenes {
+        if scene.activationState == .foregroundActive,
+           let windowScene = scene as? UIWindowScene,
+           let keyWindow = windowScene.windows.first(where: { $0.isKeyWindow })
+        {
+          return keyWindow
+        }
+      }
+      return nil
+    } else {
+      return windows.last { $0.isKeyWindow }
+    }
+  }
+
   static func topViewController(
     base: UIViewController? = UIApplication.shared.keyWindow?.rootViewController
```

Comment:
@kirillzyusko Xcode also complains about using `UIApplication.shared.keyWindow`, I haven't tested it yet but replacing this with the `activeWindow` extension removes the warning and should achieve the same thing.

<img width="945" alt="Screenshot 2024-10-09 at 2 19 47 AM" src="https://github.com/user-attachments/assets/e8890b70-9c07-44bd-aba3-878925c1d282">

Reply:
Yes @thespacemanatee I'm also constantly getting these warnings 🙈 

I would split changes in two parts anyway 🙃 Like merge these changes and then open a second PR that will repace a single line.

In this case it'll be easier to manage 😊